package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "github-action-codedeploy",
-  "version": "1.0.0",
+  "version": "1.0.1",
```

Comment:
(nit) no real need to update this version. 

But we do need this PR to go to `v1` branch and then to `master`. GHA are a little different maybe let's jump on a quick call so I can explain better and maybe we can update the README to reflect how to create/update GHAs